dadi/lib/view/send.js

```diff
@@ -1,12 +1,13 @@
-var path = require('path')
-var zlib = require('zlib')
-var brotli = require('iltorb')
-var compressible = require('compressible')
+const path = require('path')
+const zlib = require('zlib')
+const brotli = require('iltorb')
+const compressible = require('compressible')
+const CircularJSON = require('circular-json')
 
-var config = require(path.join(__dirname, '/../../../config.js'))
-var help = require(path.join(__dirname, '/../help'))
+const config = require(path.join(__dirname, '/../../../config.js'))
+const help = require(path.join(__dirname, '/../help'))
 
-var self = this
+const self = this
```

Comment:
This one should be fairly simple to remove as well, as `self` is only used once in this file.